packages/mui-material/src/FormControlLabel/FormControlLabel.test.js

```diff
@@ -331,4 +331,23 @@ describe('<FormControlLabel />', () => {
 
     expect(getByTestId('control')).to.have.attribute('value', 'test');
   });
+
+  it('label should have noWrap styles', function test() {
+    const { getByTestId } = render(
+      <FormControlLabel
+        slotProps={{
+          typography: {
+            noWrap: true,
+            'data-testid': 'typography',
+          },
+        }}
+        value="test"
+        label="Pizza"
+        control={<div data-testid="control" />}
+      />,
+    );
+    expect(getByTestId('typography')).to.have.style('white-space', 'nowrap');
+    expect(getByTestId('typography')).to.have.style('overflow', 'hidden');
+    expect(getByTestId('typography')).to.have.style('text-overflow', 'ellipsis');
```

Comment:
these are styles applied when `noWrap` is passed to typography

https://github.com/mui/material-ui/blob/9b2e523bb93a564a98696b5e68683f5a763a7acc/packages/mui-material/src/Typography/Typography.js#L81-L86